app/controllers/concerns/searchable.rb

```diff
@@ -0,0 +1,37 @@
+module Searchable
+  extend ActiveSupport::Concern
```

Comment:
Toto nie je zle, ale suhlas, ze to potrebujeme dostat do nejakeh jednotneho `Searchable` modelu aby sme vedeli robit paginate. Do toho by si sa nepustil?